cmd/namespaces.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/appuio/seiso/cfg"
+	"github.com/appuio/seiso/pkg/kubernetes"
+	"github.com/appuio/seiso/pkg/namespace"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	nsCommandLongDescription = `Sometimes Namespaces are left empty in a Kubernetes cluster.
+This command deletes Namespaces that are not being used anymore.
+A Namespace is deemed empty if no Helm releases, Pods, Deployments, StatefulSets or DaemonSets can be found.`
+)
+
+var (
+	nsCmd = &cobra.Command{
+		Use:          "namespaces",
+		Short:        "Cleans up your empty Namespaces",
+		Long:         nsCommandLongDescription,
+		Aliases:      []string{"namespace", "ns"},
+		Args:         cobra.MaximumNArgs(1),
```

Comment:
This doesn't seem to be effective nor used. If the label selector is missing, seiso exits with missing label error

Reply:
It's like this in all the other definitions as well. I would therefore leave it in. (Or shall I remove it from all the other definitions as well, e.g. [1](https://github.com/appuio/seiso/blob/master/cmd/history.go#L21), [2](https://github.com/appuio/seiso/blob/master/cmd/configmaps.go#L26), [3](https://github.com/appuio/seiso/blob/master/cmd/common.go), …)

Reply:
Well, the history command requires an additional arg, see the usage documentation `history [NAMESPACE/IMAGE]`. The configmap and secrets do not seem to use it. So I would remove it where the args are not needed.